index.html

```diff
@@ -4,54 +4,60 @@
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <link type="text/css" rel="stylesheet" href="assets/css/materialize.min.css"  media="screen,projection"/>
       <link rel="stylesheet" href="assets/css/style.css">
+      <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
```

Comment:
We are using Montserat or Poppins for the entire page. Always keep fonts consistent throughout the project.
Please comment even the smallest things like export of fonts